src/index.ts

```diff
@@ -26,6 +26,10 @@ const init: tsModule.server.PluginModuleFactory = ({ typescript: ts }) => {
   function create(
     info: tsModule.server.PluginCreateInfo,
   ): tsModule.LanguageService {
+    if (process.env.TS_PLUGIN_CSS_MODULES_DISALBED !== undefined) {
+      return info.languageService;
+    }
```

Comment:
I have basically 0 experience with the TS language server, so I don't have a ton of knowledge re: whether this is the right thing to return early with. this was just copied from the eslint plugin I referenced in the PR description.